packages/airnode-deployer/test/snapshots/index.ts

```diff
@@ -0,0 +1,33 @@
+// Manual snapshots
+
+// Ignoring the last column due to the length and content variability
+export const listAirnodes01 = `┌───────────────┬─────────────────┬───────────────────┬───────┬─────────────────┬
+│ Deployment ID │ Cloud provider  │ Airnode address   │ Stage │ Airnode version │ Last update
```

Comment:
Why is the "Last update" column emopty? Shouldn't there be at least the creation date?

Reply:
Read the comment:
`Ignoring the last column due to the length and content variability`
Can't really compare the date as its content (and therefore length) is dependent on the timezone the test is run in.